test/inductor/test_cpu_cpp_wrapper.py

```diff
@@ -281,37 +292,37 @@ class BaseTest(NamedTuple):
             "test_qconv2d",
             "cpu",
             test_mkldnn_pattern_matcher.TestPatternMatcher(),
-            condition=torch.backends.mkldnn.is_available(),
+            condition=torch.backends.mkldnn.is_available() and run_not_on_Windows(),
```

Comment:
simpler enough to just do `not IS_WINDOWS`?

Reply:
@jgong5 I use function is order to contains `TODO` comments for further turn on.